src/schemas/21-526EZ/schema.js

```diff
@@ -42,77 +63,106 @@ const addressDef = {
   type: 'object',
   oneOf: countryStateProperties, // holds country and state
   properties: {
+    // These validations (including regular expressions) come from the swagger docs
+    // Should these patterns start with ^ to make sure they apply to the whole string?
     addressLine1: {
-      type: 'string'
-      // maxLength: 51 // Until we know otherwise?
+      type: 'string',
+      maxLength: 30, // Is that going to be long enough?
+      // I think the + here means it's required
+      pattern: "([a-zA-Z0-9\-'.,,&#]([a-zA-Z0-9\-'.,,&# ])?)+$"
```

Comment:
Do these clauses both include comma twice? Also, is the second optional clause the same as the first? If so, can we simply use the first, and then the "+" will allow as many such clauses?

Reply:
Yeah, so this is a weird one.
1) Yup, double comma; not sure there's a reason
2) The second clause _is_ the same as the first except that it _also_ includes a space in the character list
    - This is to allow spaces in the regex as a whole, but not for the first character in the string (assuming we want to add that `^` in there, of course)

I'd like to change this up, but I don't want to cause a disparity between our validation and EVSS', y'know? This regex is copied directly from the swagger docs.

Reply:
Thanks for clarifying.